src/main/java/board/BoardView.java

```diff
@@ -153,9 +191,80 @@ public void actionPerformed(ActionEvent e) {
         timer.start();
     }
 
+
```

Comment:
You have a lot of unnecessary whitespace. 
It makes the code ugly.

Reply:
Hello,method delay() (inside this method has this timer.start() ) is your method i have not make any change what do you want from me to this problem??